sdks/java/harness/src/main/java/org/apache/beam/fn/harness/control/ExecutionStateSampler.java

```diff
@@ -250,6 +250,8 @@ public class ExecutionStateTracker implements BundleProgressReporter {
     private final AtomicReference<@Nullable Thread> trackedThread;
     // Read by multiple threads, read and written by the ExecutionStateSampler thread lazily.
     private final AtomicLong lastTransitionTime;
+    // Used to throttle lull logging.
+    private long lastLullReport;
```

Comment:
can you make it clearer if this is a duration or a time?

Reply:
Done.